### DISCUSSION TITLE
Data visualisation

### DISCUSSION THREAD

**Discussion by kelliedesigner** [ministryofjustice/moj-frontend#210] [EXPERIMENTAL: Components, pages and patterns]
### Existing components:

- [Data item](https://digital-land-design.herokuapp.com/component/data-item) in the Digital Land Design System by Ministry of Housing, Communities and Local Government
- [Data](https://dvsa-front-end.herokuapp.com/library/content/data) in DVSA styleguide
- [Data visualisation](https://govuk-elements.herokuapp.com/data/#data-visualisation) in GOV.UK Elements (was not brought through to the GOV.UK Design System)
- [Introduction to data visualisation](https://gss.civilservice.gov.uk/policy-store/introduction-to-data-visualisation/) - on Government Statistical Service

### Prototypes:

**Managing Offenders in Custody (prison)** - https://hmpps-moic-staging.herokuapp.com

Username: MOIC
Password: elephants

|   |   |
|---|---|
|![hmpps-moic-staging herokuapp com_handover_oversight_prison-handover-overview-tabs](https://user-images.githubusercontent.com/6122118/127132917-34985522-4586-4edb-ae36-077a8c3b19e3.png) |![hmpps-moic-staging herokuapp com_team-management_view-of-pom-overview-v2](https://user-images.githubusercontent.com/6122118/127132947-02bf2244-d86f-4138-8332-1b54da50e2ce.png) |

This product is used by staff in prisons and it allows them to:

- allocate prisoners to the appropriate Prison Offender Manager (or POM)
- view accurate caseloads to manage workload
- manage the handover of cases from prison into the community, for example, before a prisoner is released from custody

**Manage a Workforce (probation)** - https://hmpps-workforce-prototype.herokuapp.com/

Username: allocations
Password: digital

|   |   |
|---|---|
|![hmpps-workforce-prototype herokuapp com__mvp_officer-view_N57A01](https://user-images.githubusercontent.com/6122118/127133138-35511f1f-5b21-45d2-9eee-3139cd7fd938.png) |![hmpps-workforce-prototype herokuapp com__mvp_officer-view_N57A06](https://user-images.githubusercontent.com/6122118/127133172-d2acb21d-c707-4fa8-ad4c-f0da4f21846c.png) |

This product is used by probation staff in court and it allows them to:

- allocate service users to the appropriate Probation Officer in less time
- have greater insights and improve accuracy by having all the required information in one place
- spend more time on the rehabilitation needs of People on Probation

### GitHub discussions:

- [Data](https://github.com/alphagov/govuk-design-system-backlog/issues/16) GitHub issues on the GOV.UK Design System backlog

**Comment by ALmanCoder** [comment]
谢谢你朋友！
